index.html

```diff
@@ -1101,38 +1102,62 @@ <h4>
     <p>
       This section summarizes the security policies which are specified as
       normative requirements in the respective algorithms of this
-      specification:
+      specification.
     </p>
-    <ul>
-      <li>
+    <section> <h4>Secure Context</h4>
+      <p>
         Only <a>secure contexts</a> are allowed to access <a>NFC content</a>.
         Browsers may ignore this rule for development purposes only.
-      </li>
-      <li>
-        In order to use NFC, the {{Window}} object associated
-        with the {{Document}} contained by the
-        <a>top-level browsing context</a> using the Web NFC API must be
-        <a data-cite="page-visibility/#dom-visibilitystate-visible">visible</a>
-        and in <a data-cite="html#focused">focus</a>.
+      </p>
+    </section>
+    <section> <h4>Visible and focused document</h4>
+      <p>
+        Web NFC API is only avaliable for the {{Document}} of the
+        <a>top-level browsing context</a>, which must be <a>visible and focused</a>.
```

Comment:
I suggest removing "only", it works without it.

Reply:
Fine, or maybe we can use "NFC Content is only available for ..."

Reply:
Or just say "Web NFC functionality is allowed only for the ..."
then the algorithms will specify the steps on how that is handled.

Reply:
OK, done.